Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[workspace]
+resolver = "2"
+members = [ "effector", "types", "cid", "imports" ]
+
+  [workspace.dependencies]
```

Comment:
interesting formating

Reply:
yeah, the file was generated automatically

But I want the old format back, have no idea why it was changed